frontend/src/pages/projects/screens/detail/ProjectDetails.tsx

```diff
@@ -41,7 +40,7 @@ const ProjectDetails: React.FC = () => {
   const projectSharingEnabled = useIsAreaAvailable(SupportedArea.DS_PROJECTS_PERMISSIONS).status;
   const pipelinesEnabled = useIsAreaAvailable(SupportedArea.DS_PIPELINES).status;
   const modelServingEnabled = useModelServingEnabled();
-  const queryParams = useQueryParams();
+  const [queryParams, setSearchParams] = useSearchParams();
```

Comment:
Maybe change to searchParams?

```suggestion
  const [searchParams, setSearchParams] = useSearchParams();
```

Reply:
A fair point -- got me wondering what was different in the `useQueryParams` and I am not sure why we want that one lol